packages/pluggableWidgets/sidebar-web/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+All notable changes to this widget will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Added
+- We have added different toggle modes.
+- We have added dimension widget configurations for the sidebar width in expanded and collapsed state.
```

Comment:
Don't think we need to be so detailed for the first released, but we can change that later